src/model/liveposition.js

```diff
@@ -80,7 +80,9 @@ export default class LivePosition extends Position {
 	 * @returns {Boolean}
 	 */
 	is( type ) {
-		return type == 'livePosition' || type == 'model:livePosition' || super.is( type );
+		return type == 'livePosition' || type == 'model:livePosition' ||
```

Comment:
I can see that you were replacing `==` with `===` in some places, so let's be consistent and have it in all `is()` methods.

Reply:
Still not resolved.

Reply:
I'll handle that, though.